test/finsupp_notation.lean

```diff
@@ -18,3 +19,13 @@ info:
   guard <|
```

Comment:
What does `guard` and `guardMsg` mean?

Reply:
`guard` means "error if the (decidable) proposition is false". `#guard_msgs` means "error if the output is not the same as the doc-comment above".